src/components/RegistrationForm.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+
+
+export default class RegistrationForm extends Component {
+  constructor( props) {
+    super ( props )
+    this.state = {
+      name: '',
+      email: '',
+      password: '',
+      phone_number: '',
+      company: ''}
+  }
+
+  handleInput = ( event, stateKey ) => {
+    this.setState({
+      name: event.target.fullName,
+      email: event.target.email,
+      password: event.target.password,
+      phone_number: event.target.number,
+      company: event.target.company,
+      newsletter_subscribed: true
+     })
+  }
+
+  render() {
+    return (
+      <form action="http://localhost:5000/api/v1/users/register" method="post">
```

Comment:
Please generalize the use of the base url - this form will not post to localhost:5000 in other environments.